### PR TITLE
Fixes #7283

### LINKED ISSUE #7283: Knight Armor crate breaks the cargo shuttle
Github PR said it should be in security, but it was in the supplies section instead.

Also? 
**The cargo shuttle seems to break after Knight Crates are ordered, no more crates come through**
There's an extra listing that costs 0$ to buy on the supply menu, but it has no name to click on and therefore can't be ordered.

### PR DESCRIPTION
**Pull request by doobiedoo23**
This should fix the issue of knight armour absolutely mangling the cargo shuttle. If it doesn't, uh, I'm dumb

### CODE REVIEW

**Comment by Imagebard** [comment]
Lets hope this gets merged before the current round ends, honk.

**Comment by doobiedoo23** [comment]
I can't even join for some reason, was getting the connection died error every time I tried using my test server and now I'm getting it on the main server

edit: disregard this i'm dumb

**Comment by clusterfack** [comment]
It's also lacking a containertype

**Comment by PJB3005** [comment]
@clusterfack that's probably inherited?

**Comment by Imagebard** [comment]
Better safe than sorry

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -528,7 +528,7 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	access = access_armory
 	group = "Security"
 
-/datum/supply_packs/security/armory/knight //why seperate them
+/datum/supply_packs/security/armory/knight
```
Comment (clusterfack) [inline code/datums/supplypacks.dm]: Wait this typepath, does
/datum/supply_packs/security
/datum/supply_packs/security/armory
both actually exist?

Comment (PJB3005) [inline code/datums/supplypacks.dm:531]: _Suspects shoddy broken port from other codebase._

Comment (doobiedoo23) [inline code/datums/supplypacks.dm:531]: @PJB3005 yeah, you caught me, i'm a shit coder and i have no idea how to fix any of this
might be best to just revert anything i added

**Comment by clusterfack** [comment]
Don't beat yourself up about it, everyone sucks when they start

**Pull request merged**